images/macos/provision/core/node.sh

```diff
@@ -1,10 +1,5 @@
 source ~/utils/utils.sh
 
-node_common_modules=(
-  node-gyp
-  mobile-center-cli
-)
-
 node_catalina_modules=(
```

Comment:
let's change the name since we're going to install these modules for Big Sur as well

Reply:
Thanks!